src/ILCompiler.MetadataTransform/tests/SingleFileMetadataPolicy.cs

```diff
@@ -3,13 +3,21 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Reflection;
 using ILCompiler.Metadata;
 using Internal.TypeSystem;
 
 namespace MetadataTransformTests
 {
     struct SingleFileMetadataPolicy : IMetadataPolicy
     {
+        ExplicitScopeAssemblyPolicyMixin _explicitScopePolicyMixin;
+
+        public void Init()
+        {
+            _explicitScopePolicyMixin = new ExplicitScopeAssemblyPolicyMixin();
```

Comment:
Nit: Since `ExplicitScopeAssemblyPolicyMixin` is a class, it should be pretty easy to make it lazy initialized and we could avoid all the ugly `Init` calls.